src/app/audiogram/audiogram.component.ts

```diff
@@ -58,7 +58,7 @@ export class AudiogramComponent implements OnInit {
   }
   public severityChange(event: MatRadioChange) {
     let sevs: Array<string> = ['leftHighSev', 'leftLowSev', 'rightHighSev', 'rightLowSev'];
-    if(sevs.includes(event.source.name)) {
+    if(sevs.indexOf(event.source.name) > -1) {
```

Comment:
What was wrong with `.includes`? Seems more idiomatic that way.

Reply:
For this one, I was getting an error when I'd compile saying that includes was not defined for an array of strings. I was able to make it work with that much less informative expression.

Reply:
https://github.com/movshov/VA-Audiology-App/pull/25 prevents this, go ahead and switch it back to `includes`

Reply:
Okay will do